integration_tests/test_alascan.py

```diff
@@ -17,7 +17,7 @@ def alascan_module():
     """Return a default alascan module."""
     with tempfile.TemporaryDirectory(dir=".") as tmpdir:
         alascan = AlascanModule(
-            order=0, path=".", initial_params=DEFAULT_ALASCAN_CONFIG
+            order=1, path=Path("."), initial_params=DEFAULT_ALASCAN_CONFIG
```

Comment:
Why do you need to change the order here? Also in the other tests

Reply:
This is an issue related to this new implementation, and the inheritance of `AlascanModule` <- `HaddockModule.__init__()`
- If the order is `0` (first module in the workflow): It will search for input molecules stored somewhere that is not created for this integration test and `[src/haddock/libs/libontology.py](https://github.com/haddocking/haddock3/pull/921/files/38bb99b82ddf685e8f56d2da13e007821cbf6638#diff-15b98e8f9e924d3f27cf4aa6d3afebfcea090bc0a6594271706e7e134e3db4b2) line 279 will trigger a crash.
- If not, it will not search for it, and all fine